### PR TITLE
Incorporating changes from reviews of past pull request - Feature psi

### PR DESCRIPTION
**Pull request by powerlink-team-kalycito**
[TASK] Incorporated the review comments of pull request 2

- Coding guidelines related comments incorporated.
- Unnecessary sections in cmake files has been removed.
- In oplkcfg.h the sections are removed since it is not used.
- In logbook.c and tssdo.c kErrorSdoUdpArpInProgress comment.
  has been removed since the arp query feature is implemented
  in the openPOWERLINK V2.5 stack.
- White space comments incorporated

Change-Id: If2bbf238f8262129b1bd47aeedbde6f73e2af2ee

### CODE REVIEW

**Comment by zelenkaj** [comment]
No need to wait for me, just a comment to the pull request in general: Please follow the same flow with pull requests like we agreed for oplk!

File: blackchannel/POWERLINK/pcp/psi/include/psi/obdict.h [hunk 1]
```diff
@@ -60,13 +60,11 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 // function prototypes
 //------------------------------------------------------------------------------
 #if(((PSI_MODULE_INTEGRATION) & (PSI_MODULE_CC)) != 0)
-  extern tOplkError cc_obdAccessCb(tObdCbParam* pParam_p);
+extern tOplkError cc_obdAccessCb(tObdCbParam* pParam_p);
 #endif
 
 #if(((PSI_MODULE_INTEGRATION) & (PSI_MODULE_SSDO)) != 0)
-  extern tOplkError rssdo_obdAccessCb(tObdCbParam* pParam_p);
+extern tOplkError rssdo_obdAccessCb(tObdCbParam* pParam_p);
```
Comment (powerlink-team-kalycito) [inline blackchannel/POWERLINK/pcp/psi/include/psi/obdict.h]: Okay.

File: blackchannel/POWERLINK/pcp/psi/include/psi/obdict.h [hunk 2]
```diff
@@ -60,13 +60,11 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 // function prototypes
 //------------------------------------------------------------------------------
 #if(((PSI_MODULE_INTEGRATION) & (PSI_MODULE_CC)) != 0)
-  extern tOplkError cc_obdAccessCb(tObdCbParam* pParam_p);
+extern tOplkError cc_obdAccessCb(tObdCbParam* pParam_p);
```
Comment (powerlink-team-kalycito) [inline blackchannel/POWERLINK/pcp/psi/include/psi/obdict.h]: Okay.

File: blackchannel/POWERLINK/pcp/psi/CMakeLists.txt [hunk 3]
```diff
@@ -158,13 +157,7 @@ SET(OMETHLIB_INCS
...
    )
 
-SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -DCONFIG_DLL_PRES_CHAINING_CN=TRUE -DNMT_MAX_NODE_ID=254 -DCONFIG_INCLUDE_PDO -DCONFIG_INCLUDE_SDO_ASND -DCONFIG_INCLUDE_MASND")
-ADD_DEFINITIONS(-DCONFIG_DLL_PRES_CHAINING_CN)
-ADD_DEFINITIONS(-DNMT_MAX_NODE_ID=254)
-ADD_DEFINITIONS(-DCONFIG_INCLUDE_PDO)
-ADD_DEFINITIONS(-DCONFIG_INCLUDE_SDO_ASND)
-ADD_DEFINITIONS(-DCONFIG_INCLUDE_MASND)
-ADD_DEFINITIONS(-DCONFIG_INCLUDE_SDO_UDP)
+SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -DCONFIG_DLL_PRES_CHAINING_CN=TRUE -DNMT_MAX_NODE_ID=254 -DCONFIG_INCLUDE_PDO -DCONFIG_INCLUDE_SDO_ASND -DCONFIG_INCLUDE_MASND -DCONFIG_INCLUDE_SDO_UDP")
```
Comment (powerlink-team-kalycito) [inline blackchannel/POWERLINK/pcp/psi/CMakeLists.txt]: CONFIG_INCLUDE_SDO_UDP is also defined in blackchannel\POWERLINK\pcp\psi\oplkcfg.h hence it shall be removed.

**Pull request closed without merging**